Sources/PrimerSDK/Classes/Data Models/PaymentMethodConfigurationOptions.swift

```diff
@@ -26,7 +26,31 @@ struct CardOptions: PaymentMethodOptions {
 struct MerchantOptions: PaymentMethodOptions {
     let merchantId: String
     let merchantAccountId: String
-    let appId: String? // Nol pay, Klarna
+    let appId: String? // Nol pay
```

Comment:
Can we take a look at the formatting here, add a newline between the structures, its all a bit squashed

Reply:
For some reason lint issues forced me to remove the new lines. But i'll re add them.

Reply:
Newlines should be fine, in case you are being tripped up by spaces, check your xcode settings. Happy to jump on a huddle if something isnt working and compare setups